### ISSUE TITLE
[Bug] Markdown Monster shim creating issue (@2.6.7)

### ISSUE THREAD

**Issue by axtens** [ScoopInstaller/Extras#8933]
#### Current Behavior
Markdown Monster fails to update due to shim creation not finding an exe
```
C:\Users\bugma\Source\Repos\Present\Scripts\siteMapping>scoop update markdown-monster
markdown-monster: 2.6.6 -> 2.6.7
Updating one outdated app:
Updating 'markdown-monster' (2.6.6 -> 2.6.7)
Downloading new version
MarkdownMonsterPortable.zip (25.8 MB) [=======================================================================] 100%
Checking hash of MarkdownMonsterPortable.zip ... ok.
Uninstalling 'markdown-monster' (2.6.6)
Unlinking ~\scoop\apps\markdown-monster\current
Installing 'markdown-monster' (2.6.7) [64bit]
Loading MarkdownMonsterPortable.zip from cache
Extracting MarkdownMonsterPortable.zip ... done.
Linking ~\scoop\apps\markdown-monster\current => ~\scoop\apps\markdown-monster\2.6.7
Creating shim for 'mmcli'.
Can't shim 'mmcli.exe': File doesn't exist.
```

**Comment by niheaven** [comment]
Markdown Monster v2.6.7 doesn't bundle `mm.exe` and `mmcli.exe`, so wait for upsteam fixes.

**Comment by tech189** [comment]
Fixed in version 2.6.8:
```
Checking hash of MarkdownMonsterPortable.zip ... ok.
Extracting MarkdownMonsterPortable.zip ... done.
Linking ~\scoop\apps\markdown-monster\current => ~\scoop\apps\markdown-monster\2.6.8
Creating shim for 'mmcli'.
Creating shortcut for Markdown Monster (MarkdownMonster.exe)
Persisting PortableSettings
'markdown-monster' (2.6.8) was installed successfully!
```
